### ISSUE TITLE
Gist to plugins for Gradle helper functions

### ISSUE THREAD

**Issue by hypest** [wordpress-mobile/gutenberg-mobile#334]
A few Gradle helper functions/utils (for the Android side of things) have been recently developed that are published as GitHub gists. We might want to make one or more Gradle plugins and host them in a proper repo instead.

Helper functions:
* [submoduleGitHash()](https://gist.github.com/hypest/e06f6097065728b6db7b7c462f8fef1a)
* [readReackNativeVersion()](https://gist.github.com/hypest/742448b9588b3a0aa580a5e80ae95bdf)
* [waitJitpack()](https://gist.github.com/hypest/f526fe0775dedce0ce0133f1400d22a4)

**Comment by SergioEstevao** [comment]
@hypest is this something that we still aim to do?

**Comment by hypest** [comment]
Yes, I think that depending on personal gists is not clean or maintainable. The code should be tracked in some proper form, although it's not a high priority apparently.
